source/index-management.rst

```diff
@@ -71,7 +71,18 @@ All drivers MUST offer at least one of the sections of operations, the Standard
 Operation Parameters
 --------------------
 
-All drivers MUST include the specified parameters in each operation, with the exception of the options parameter which is OPTIONAL. As of 3.4 (see https://jira.mongodb.org/browse/SERVER-769) the server validates options passed to the createIndexes command -- drivers should be aware when testing that passing arbitrary options when the driver does not validate them could fail on the server.
+All drivers MUST include the specified parameters in each operation. This does not preclude a driver from offering more. A driver SHOULD NOT require a user to specify the options parameter if they wish to use the server defaults.
```

Comment:
Let's work really hard to avoid `SHOULD` ever sneaking into our specs again. Do we have to include this caveat if the API notes that a given property is `Optional`? I'm guessing this came up due to [SERVER-47193](https://jira.mongodb.org/browse/SERVER-47193)?